docs/src/guide/parallelization.md

```diff
@@ -62,8 +59,18 @@ as a breakdown over individual routines.
     unless you set `DFTK_TIMING` to `"all"`. In this case you must not use
     Julia threading (see section below) or otherwise undefined behaviour results.
 
+## MPI
+DFTK uses MPI to distribute on kpoints only at the moment. This should be the most performant method of parallelization: if you have kpoints, start by disabling all threading and use this. Simply follow the instructions on [MPI.jl](https://github.com/JuliaParallel/MPI.jl) and run DFTK under MPI:
```

Comment:
Too long a line.

Reply:
That's OK for docs, no?

Reply:
Not in my world ... docs are code, too and should be easy to read in an editor ... just do your emacs magic to break the line and I'm happy :smile:

Reply:
Just fix your vim already :-) but ok done